code/modules/atmos_automation/console.dm

```diff
@@ -292,9 +292,10 @@
 		onclose(usr, "AAC_assemblies")
 
 /obj/machinery/computer/general_air_control/atmos_automation/proc/AAC_age_check(var/mob/M,var/list/href_list)
-	if(href_list["on"] || href_list["runonce"])
-		if(M.client && !M.client.holder && M.client.player_age < 30)
-			message_admins("[key_name(M)] attempted to [href_list["on"] ? "toggle" : "single-run"] an AAC script despite their player age of [M.client.player_age].")
+	if(href_list["read"])
+		if(M.client && !M.client.holder && M.client.player_age < 7)
+			to_chat(M, "<span class=warning>Code importing functionality is not available yet. Keep playing to unlock it.</span>")
```

Comment:
Kinda weird to explicitly mention the player while IC.
I'd change it to "Script importing whitelist pending... Come back in [7 - M.client.player_age] days..." or "Script Import functionality disabled for new employees. Come back in [7 - M.client.player_age] days."

Reply:
i fear that making it more IC might make the reason more ambiguous to a player as to why they can't do it.

Reply:
I'd say that having the reason IC with a "come back in SOME days" blurb is obvious enough, and it's not gonna be that big an issue for an average newplayer, cause the newlad wouldn't really have access to scripts for the AAC yet, unless another player shared it to him that very same round (or previous one).